SB2018/2018-03-09.md

```diff
@@ -0,0 +1,22 @@
+---
+to: All Teams
+subject: New SourceBots Kit Update
+---
+
+Hi!
+
+We're happy to announce a new update to our kits!
+
+This release contains many improvements to vision (other changes too):
+- Completely rewrite vision to greatly increase its accuracy (now all Cartesian values have an error of only 15% of their total distance).
```

Comment:
Perhaps describing this as a "tolerance" would be a better phrasing than "error"?